storage/innobase/xtrabackup/src/read_filt.cc

```diff
@@ -249,8 +247,9 @@ static void rf_page_tracking_get_next_batch(xb_fil_cur_t *cursor,
    * backup start_lsn instead of last backup end_lsn.
    * We read all pages by setting read_batch_len to the size of file */
 
-  /* if DDL happened on the table after the checkpoint LSN we do full scan on
-  that table. index_load_map is poped during the first scan of redo */
+  /* if inplace DDL having MLOG_ADD_INDEX happened on the table after the
```

Comment:
If INPLACE DDLs that generate MLOG_INDEX_LOAD happen on table after the checkpoint.."

Reply:
fixed